content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -0,0 +1,109 @@
+ * Progress note: Continue diet and exercise to achieve optimal weight and goal achievement of wearing uniform for upcoming reunion.    
+
+  
+Soarian Clinicals has a robust capability for clients to configure their clinical charting content, format, and structure as appropriate to their practice; therefore, the API response will depend on that configuration. While the resource structure supports specific relationships between data types, that structure cannot be enforced without restricting client practice or risking inaccurate or incomplete information. Therefore to preserve completeness and accuracy per clinically charted content, clients configure the data relevant to this resource and the API will return charted content, not marked as erroneous, for the elements the client has configured. To preserve accuracy and completeness, conceptual data types of Concerns, Goals, Assessments and Plans will all be returned together as a CarePlan response. Data is returned in reverse chronological order by most current charted date for qualifying encounters in the sequence defined in configuration.  
+
+Note that this information is documented during and in context of an encounter, may not be actively managed, and may not be consolidated across encounters as a "patient-level" plan. Therefore, this data should be interpreted as a snapshot in time of a point in time. For example, if a patient has an encounter related to injury due to a car accident, there will be a snapshot of this information in context of those injuries. If there is a subsequent encounter related to an illness, there may be a snapshot of this data set related to that illness. If a patient queries for the care plan resource a week after the last encounter, the response will be the latest instance of the last set of this data documented; i.e., the most recent charted version of the illness related data. If the query includes a date range inclusive of both encounters, the response will include the most recent charted version of both the injury and illness data sets. Both data sets will be the most recent data documented for the encounter, not of a specific dated instance, and, since in practice, they may not be actively managed, they may not reflect the current state of goals, concerns and plans; they may represent the initially documented state. For context, the narrative response will provide a charted date to represent to what date context this information is relevant.
+
+
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+
+## Search
+	
+	GET /CarePlan?:parameters
+	
+_Implementation Notes_
+
+* The API returns the most current careplan list.
+
+* The API returns values in reverse chronological order by the most currently charted date.
+
+* The API returns charted content in active status.
+
+* The API returns charted content that is not marked as erroneous.
```

Comment:
Remove this. Clarify sentence above. Are only status='active' returned? What about proposed, draft, or completed? I assume cancelled is the erroneous status you refer to. If so, call that out (eg: It will not return a CarePlan with a cancelled status)